tests/cc13d/input.dat

```diff
@@ -0,0 +1,25 @@
+memory 500 mb
+
+molecule water{
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+  reference RHF
+  basis cc-pVDZ
+  dertype first
```

Comment:
Sure, I can get rid of 'dertype' as it is indeed superfluous. will remove the set memory option too. thanks for the feedback.